Source/Generic/ECTestCase.m

```diff
@@ -719,19 +720,19 @@ - (BOOL)image:(NSBitmapImageRep*)image matchesReferenceImage:(NSBitmapImageRep*)
 	CGFloat pixelThreshold = [mergedProperties[@"pixelThreshold"] doubleValue];
 	NSSize maxSize = NSZeroSize;
 	if ([mergedProperties[@"maxSizeMatchesReference"] boolValue]) {
-		maxSize = reference.size;
+		maxSize = NSMakeSize(reference.pixelsWide, reference.pixelsHigh);
 	} else {
 		maxSize = NSMakeSize([mergedProperties[@"maxWidth"] doubleValue], [mergedProperties[@"maxHeight"] doubleValue]);
 	}
 
-	NSSize imageSize = image.size;
+	NSSize imageSize = NSMakeSize(image.pixelsWide, image.pixelsHigh);
 	if ((imageSize.width > maxSize.width) || (imageSize.height > maxSize.height))
 	{
 		NSLog(@"image looks a bit big: %@", NSStringFromSize(imageSize));
 		return NO;
 	}
 
-	NSSize referenceSize = reference.size;
+	NSSize referenceSize = NSMakeSize(reference.pixelsWide, reference.pixelsHigh);
```

Comment:
I'm curious, why these changes? Why is it better to use `pixelsWide` etc. rather than `size`?